src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToStartHandlerTest.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.caseupdated;
+
+import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType.ABOUT_TO_START;
+
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Appeal;
+import uk.gov.hmcts.reform.sscs.ccd.domain.CaseDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DynamicList;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DynamicListItem;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingOptions;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.util.CaseDataUtils;
+import uk.gov.hmcts.reform.sscs.reference.data.model.Language;
+import uk.gov.hmcts.reform.sscs.reference.data.service.VerbalLanguagesService;
+import uk.gov.hmcts.reform.sscs.util.DynamicListLanguageUtil;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CaseUpdatedAboutToStartHandlerTest {
+
+    private static final String USER_AUTHORISATION = "Bearer token";
+
+    @Mock
+    private Callback<SscsCaseData> callback;
+
+    @Mock
+    private CaseDetails<SscsCaseData> caseDetails;
+
+    @Mock
+    private SscsCaseData sscsCaseData;
+
+    @Mock
+    private DynamicListLanguageUtil dynamicListLanguageUtil;
+
+    @Mock
+    private VerbalLanguagesService verbalLanguagesService;
+
+    @InjectMocks
+    private CaseUpdatedAboutToStartHandler handler;
+
+    @Before
+    public void setUp() {
+        sscsCaseData = SscsCaseData.builder().appeal(Appeal.builder().build()).build();
```

Comment:
Could separate the builders out, so Appeal appeal = Appeal.buil...etc then set the .appeal(appeal) for caseData. General rule to keep these separate and not nest them (even when they're simple/empty like this).